docs/basics/userguide/export_training_data.md

```diff
@@ -0,0 +1,162 @@
+[INFO] 16:01: [PPO[worker: 0]] | max_global_step = 51200 | time/iterations = 24 | rollout/ep_rew_mean = 370.78 | rollout/ep_len_mean = 370.78 | time/fps = 322 | time/time_elapsed = 152 | time/total_timesteps = 49152 | train/learning_rate = 0.0003 | train/entropy_loss = -0.49728800179436805 | train/policy_gradient_loss = -0.002779247868602397 | train/value_loss = 0.04587990254440229 | train/approx_kl = 0.005585251376032829 | train/clip_fraction = 0.0578125 | train/loss = 0.01021644752472639 | train/explained_variance = 0.028099477291107178 | train/n_updates = 230 | train/clip_range = 0.2 |
+[INFO] 16:01: ... trained!
+[INFO] 16:01: Saved ExperimentManager(PPO) using pickle.
+[INFO] 16:01: The ExperimentManager was saved in : 'rlberry_data/temp/manager_data/PPO_2024-06-28_15-58-48_4fc693bc/manager_obj.pickle'
+Backend tkagg is interactive backend. Turning interactive mode on.
+```
+
+
+Instead using {mod}`rlberry.manager.plot_writer_data` as in the [visualization page](visu_userguide), you can extract the information into dataframe to use it with other tools.
+To get these data, use the {mod}`rlberry.manager.read_writer_data` function.
+
+```python
+df = read_writer_data([manager])
+```
+
+Then, you can use your own tools to display whatever you like.
+
+
+In this example, it will be the rewards through the training of the 3 `fit` with matplotlib, but up to you display other information, use other tools, etc.
+
+<span>&#9888;</span> Of course, the information contained in the writer depends on how it has been configured and what the agent has recorded in it.<span>&#9888;</span>
+
+```python
+figure, ax = plt.subplots(1, 1)
+
+toplot_df1 = df.loc[
+    (df["tag"] == "rollout/ep_rew_mean") & (df["n_simu"] == 0), ["global_step", "value"]
+]
+toplot_df2 = df.loc[
+    (df["tag"] == "rollout/ep_rew_mean") & (df["n_simu"] == 1), ["global_step", "value"]
+]
+toplot_df3 = df.loc[
+    (df["tag"] == "rollout/ep_rew_mean") & (df["n_simu"] == 2), ["global_step", "value"]
+]
+
+
+ax.plot(toplot_df1["global_step"], toplot_df1["value"])
+ax.plot(toplot_df2["global_step"], toplot_df2["value"])
+ax.plot(toplot_df3["global_step"], toplot_df3["value"])
+
+ax.set_xlabel("steps")
+ax.set_ylabel("rewards")
+plt.show()
+```
+
+![image](read_writer_example.png)
+
+
+In this previous example, {mod}`rlberry.manager.read_writer_data` had an [ExperimentManager](rlberry.manager.ExperimentManager) as `data_source`, but it can also be a list of ExperimentManager (if you need data on more than one experimentManager), or a path(String) to a directory containing pickle files of an [ExperimentManager](rlberry.manager.ExperimentManager).
```

Comment:
...but it can also be a list... -> ... but read_writer_data can also take as input a list..."